pages/booklet.tsx

```diff
@@ -34,48 +36,85 @@ const Booklet: React.FC = () => {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <ToggleHeader />
-      <main id="purchase">
-        <div className="book-description">
-          <div className="book-image">
-            <Image className="book-front-cover" src={BookImage} alt={t("bookImageAlt")} />
-          </div>
-          <div className="book-text">
-            <h1>{t("heading")}</h1>
-            <p className="book-subtext">{t("subtext")}</p>
-            <p className="book-subtitle">{t("subtitle")}</p>
-            <h3>{t("purchase")}: </h3>
-            <div className="purchase-links">
-              <PurchaseButtons />
-              <Link href="https://amazon.com" target="_blank">
-                {t("order")}
-              </Link>
-            </div>
-          </div>
+      <main id="booklet">
+        <div className="w-100 book-description position-relative">
+          <Container>
+            <Row xs={1} sm={1} md={1} xl={2}>
+              <Col className="align-items-center">
+                <div className="book-image">
+                  <BootstrapImage
+                    className="book-front-cover"
+                    src="/photos/booklet/Book - Cover EN 2.png"
```

Comment:
Very minor comment, not a blocker. But at some point if we could standardize our photo names some thing like "book_cover_en_2.png" that'd be great